services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -660,6 +664,93 @@ testScimCreateVsUserRef = do
       (call . delete $ brig . paths ["i", "users", toByteString' uid])
         !!! const 202 === statusCode
 
+testCreateUserTimeout :: TestSpar ()
+testCreateUserTimeout = do
+  env <- ask
+
+  email <- randomEmail
+  scimUser <- randomScimUser <&> \u -> u {Scim.User.externalId = Just $ fromEmail email}
+  (_owner, tid) <- call $ createUserWithTeam (env ^. teBrig) (env ^. teGalley)
+  tok <- registerScimToken tid Nothing
+
+  testFlow tid tok scimUser email
+  -- creating the same user again should behave the same
+  testFlow tid tok scimUser email
+  where
+    testFlow tid tok scimUser email = do
+      env <- ask
+      let brig = env ^. teBrig
+
+      scimStoredUser <- createUser tok scimUser
+      let handle = Handle . Scim.User.userName . Scim.value . Scim.thing $ scimStoredUser
+
+      inv <- call $ getInvitation brig email
+
+      -- Wait for user and invitation to expire
+      let setTeamInvitationTimeout = 5 -- Must match the value in brig.integration.yaml
```

Comment:
```suggestion
      let setTeamInvitationTimeout = env ^. teOpts ...
```

ah.  no.  this is annoying.  [this is the value we're looking for](https://github.com/wireapp/wire-server/blob/84ca176fb79e498db50eb9097dcb322ceec91e82/services/brig/brig.integration.yaml#L133), but we can't access that here.

i think we have to change the `TestEnv` type to contain the bring service options next to the spar service options.  it's not a complicated change, just really annoying.

Reply:
I'll give this one a shot.

Reply:
dcb21fa6602f48e035d6ab4575930fe1874ead8b